src/util/client.js

```diff
@@ -1,181 +1,218 @@
+    })
+  }
+
+  static getAllCategoriesByMenu = menuId => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/categories-by-menu/" + menuId, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static updateMenu = (id, data) => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.put(ROOT_URL + "/menus/" + id, data, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static createMenu = data => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.post(ROOT_URL + "/menus/", data, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static deleteMenu = id => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.delete(ROOT_URL + "/menus/" + id, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static getMenu = id => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/menus/" + id, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static getAllMenus = () => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/all-menus/", {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static getTags = () => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/tags", {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static uploadMenu = data => {
+    let token = retrieveUserToken()
+    return axios.post(
+      ROOT_URL + "/uploadMenuCSV",
+      {
+        data: data,
+      },
+      {
+        headers: { Authorization: `Bearer ${token}` },
+      }
+    )
+  }
+
+  static registerRestaurant = data => {
+    return axios.post(ROOT_URL + "/restaurants/register", data)
+  }
+
+  static registerUser = data => {
+    return axios.post(ROOT_URL + "/user/register", data)
+  }
+
+  static checkEmail = email => {
+    return axios.get(ROOT_URL + "/user/check-email", {
+      params: { email: email },
+    })
+  }
+
+  static getPersonalInfo = () => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/user/details/", {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static updatePersonalInfo = (id, data) => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.put(ROOT_URL + "/user/details/" + id, data, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static getRestaurantInfo = () => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.get(ROOT_URL + "/restaurants/me/", {
+      headers: { Authorization: `Bearer ${token}` },
+    })
+  }
+
+  static updateRestaurantInfo = (id, data) => {
+    let token = retrieveUserToken() // get user auth token
+    return axios.put(ROOT_URL + "/restaurants/" + id, data, {
```

Comment:
So here it was be one arg and now two, but I don't see any changes to the usages of this method. Would that be the cause of the weird 404 problem?

Reply:
Referring to [this](https://github.com/Nomi-Technologies/menu-service/pull/44#discussion_r482632550)